docs/remote.go

```diff
@@ -21,7 +21,7 @@ const (
   standalone keyservers and OCI/Docker registry credentials through its
   subcommands.
 
-  A 'remote endpoint' is the Sylabs Cloud, an Apptainer Enterprise installation,
+  A 'remote endpoint' is the Sylabs Cloud, a Singularity Enterprise installation,
```

Comment:
Should this be removed when library and remote builder is removed?

Reply:
Yes, that would be part of removing remote builder

Reply:
deferred to removal of non-LF code